src/controllers/api/account.ts

```diff
@@ -480,14 +481,14 @@ export class AccountController {
 		// 4. Check the token balance for Testnet account
 
 		// 1. Get logTo UserId from request body
-		const { username } = request.body;
+		const { name, primaryEmail } = request.body;
```

Comment:
@DaevMithran the `createValidator` is out of sync with current implementation.

Reply:
Fixed it